standard/expressions.md

```diff
@@ -1283,6 +1285,50 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 > ```
 > Within the `A` class, those occurrences of the Color identifier that reference the Color type are delimited by `**`, and those that reference the Color field are not. *end example*
 
+### §null-conditional-member-access Null Conditional Member Access
```

Comment:
I've just been trying to figure out how this works in a couple of cases, and while I think it's the right approach, we may need a bit more explanation.

For example consider:

```csharp
string x = null;
int? y = x?.GetHashCode();
```

My understanding is that that is a null_conditional_member_access with a dependent_access of `()` - so we get the null value. It's *not* a null_conditional_invocation_expression, because that's only used in places where the result (if any) is discarded. Is that the case? If so, it's a bit confusing because `x.GetHashCode()` is an invocation expresson, isn't it?

I'm not suggesting that we change any of the existing text - just consider adding an example/note.